Update README.md (#52)
jarneson/godot-ply: README.md

<br>Alt + Clicking in edge or face mode will select loops.

And three gizmo modes:

> - Global - Translate/Rotate/Scale along with global coordinates

Reviewer: ```suggestion
- Global - Translate/Rotate/Scale along global coordinates
```

		- Invert Normals: Inverts the normals of all faces
    - Mesh Utilities
        - Export to OBJ: Exports the selected mesh to an OBJ file
            - Currently exports basic geometry, excluding normals, materials, etc
        - Quick Generators
            - Plane: Generate a two-sided unit plane
            - Cube: Generate a unit cube
        - Generate: Opens a modal for more advanced generation

>             - Plane: Generate a plane with specified size and subdivisions
>             - Cube: Generate a cube with specified size and subdivisions

Reviewer: ```suggestion
            - Plane: Generate a plane with a specified size and subdivision count
            - Cube: Generate a cube with a specified size and subdivision count
```

            - Isosphere: Generate an isosphere with a specified radius and subdivisions

>             - Cylinder: Generate a cylinder with a specified radius, depth, circle vertex count, and 

Reviewer: ```suggestion
            - Cylinder: Generate a cylinder with a specified radius, depth, and circle vertex count
```

- Face
    - Select Faces
        - ![loopicon1](./addons/ply/icons/face_loop.svg) Loop: a quad loop in one direction
        - ![loopicon2](./addons/ply/icons/face_loop_2.svg) Loop: a quad loop in the other direction
    - Face Tools
        - ![extrudeicon](./addons/ply/icons/extrude_face.svg) ` ctrl-e ` Extrude: Extrudes the selected face(s) along their mean normal by 1 unit
        - Connect: Remove the two selected faces, creating a new face between edges. Tries to choose an edge pairing that works.. but not always.
        - Subdivide: Subdivide a quad or a tri into 4 quads or 4 tris
        - Triangulate: Triangulates a face using an ear clipping algorithm

>     - Paint Faces: Moves the selected face to the chosen surface, allowing multiple materials per Ply Instance. Assign materials to the parent MeshInstance.

Reviewer: ```suggestion
    - Paint Faces: Moves the selected face to the chosen surface, allowing multiple materials per mesh. Assign materials to the parent MeshInstance or in the Ply Editor materials array.
```

    - None, yet!

### Collisions
Collision meshes are updated if there is a CollisionShape child of the parent MeshInstance node at `$StaticBody/CollsionShape`. This is the default naming if you use the `Create Trimesh Static Body` tool.

## Details
Meshes are meant to only be oriented manifolds. Some properties:
- Each edge has one or two faces (although we generally use precisely 2)
- All of an edge's faces have compatible orientation -- that is, the edge origin and destination are in the opposite order for opposite faces.

> Ply uses a winged edge representation for edges but omit counterclockwise navigation:

Reviewer: ```suggestion
Ply uses a winged edge representation for edges but omits counterclockwise navigation:
```
